libraries/lib-wave-track/WaveTrack.cpp

```diff
@@ -1905,6 +1906,7 @@ std::shared_ptr<WaveClip> WaveTrack::RemoveAndReturnClip(WaveClip* clip)
    if (it != mClips.end()) {
       auto result = std::move(*it); // Array stops owning the clip, before we shrink it
       mClips.erase(it);
+      Observer::Publisher<WaveTrackClipEvent>::Publish({});
```

Comment:
Probably this should be wrapped into some kind `NotifyClipsChanged` function or whatnot